sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -1911,6 +1934,8 @@ def test_inverses_nseries():
     # assert acot(x**2 - S(1)/2*I)._eval_nseries(x, 3, None, -1) == I*acoth(S(1)/2) - 4*x**2/3 + O(x**3)
     # assert acot(x)._eval_nseries(x, 2, None, 1) == pi/2 - x + O(x**2)
     # assert acot(x)._eval_nseries(x, 2, None, -1) == -pi/2 - x + O(x**2)
```

Comment:
There are many series like this at have been commented out in `test_trigonometric.py` from lines 1870 - 1980 . But now that we have proper `_eval_nseries` methods for all inv trig functions , I guess all of these would work (will try it out) so not sure why they have been commented out or do we need all of these ! @jksuom 

I guess we could also restructure these tests better creating separate tests per each inverse function , rather than creating a single 110 line test function

Reply:
I ran a check , all the commented tests seem to work properly !